llvm/test/CodeGen/NVPTX/atomics-with-semantics.ll

```diff
@@ -2126,6 +2126,437 @@ entry:
   ; CHECK: atom.acq_rel.cta.shared.cas.b64
   %tmp701 = tail call i64 @llvm.nvvm.atomic.cas.shared.i.acq.rel.cta.i64.p3i64(i64 addrspace(3)* %llp3, i64 %ll, i64 %ll);
 
+  ; CHECK: ld.relaxed.gpu.s32

```

Comment:
Are there tests for `ld.volatile` variants?

Reply:
They were missing. I added them now.